keyboards/gmmk/pro/config.h

```diff
@@ -39,27 +41,64 @@
 #define BOOTMAGIC_LITE_COLUMN 3
 
 #define TAP_CODE_DELAY 10
-#define ENCODERS_PAD_A { C15 }
-#define ENCODERS_PAD_B { C14 }
+#define ENCODERS_PAD_A \
+    { C15 }
+#define ENCODERS_PAD_B \
+    { C14 }
 
 /* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
 #define LOCKING_SUPPORT_ENABLE
 /* Locking resynchronize hack */
 #define LOCKING_RESYNC_ENABLE
 
-/* SPI Config for LED Driver */
 #define SPI_DRIVER SPID1
 #define SPI_SCK_PIN A5
-#define SPI_MOSI_PIN A6
-#define SPI_MISO_PIN A7
+#define SPI_SCK_PAL_MODE 5
+#define SPI_MISO_PIN A6
+#define SPI_MISO_PAL_MODE 5
+#define SPI_MOSI_PIN A7
+#define SPI_MOSI_PAL_MODE 5
 
-#define DRIVER_1_CS B13
-#define DRIVER_2_CS B14
-#define DRIVER_1_EN C13
-#define DRIVER_2_EN C13
+#define SPI_SS_DRIVER_1_PIN B13
+#define SPI_SS_DRIVER_2_PIN B14
+#define ENABLE_DRIVERS_PIN C13
+
+#define SW_LINES_ENABLE_DRIVER_1 SW_LINES_ENABLED_1_TO_12
+#define SW_LINES_ENABLE_DRIVER_2 SW_LINES_ENABLED_1_TO_12
 
 #define DRIVER_COUNT 2
 #define DRIVER_1_LED_TOTAL 66
 #define DRIVER_2_LED_TOTAL 32
 #define DRIVER_LED_TOTAL (DRIVER_1_LED_TOTAL + DRIVER_2_LED_TOTAL)
+#define RGB_MATRIX_CENTER \
+    { 112, 32 }
+#define RGB_MATRIX_MAXIMUM_BRIGHTNESS 175
```

Comment:
This should be left as default, use a different define to control the current scaling factor you write in `AW20216S_init()`